src/ClientBuilder.php

```diff
@@ -231,13 +241,39 @@ public function setRepresentationSerializer(RepresentationSerializerInterface $r
     /**
      * {@inheritdoc}
      */
-    public function setSdkIdentifierSuffix(?string $sdkIdentifierSuffix): void
+    public function setSdkIdentifier(string $sdkIdentifier): void
     {
-        $this->sdkIdentifier = Client::SDK_IDENTIFIER;
+        $this->sdkIdentifier = $sdkIdentifier;
+    }
 
-        if ($sdkIdentifierSuffix) {
-            $this->sdkIdentifier .= '.' . $sdkIdentifierSuffix;
+    /**
+     * @return string
+     */
+    private function getSdkVersion(): string
+    {
+        if (null === $this->sdkVersion) {
+            $this->setSdkVersionByPackageName('sentry/sentry');
         }
+
+        return $this->sdkVersion;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setSdkVersion(string $sdkVersion): void
+    {
+        $this->sdkVersion = $sdkVersion;
+    }
+
+    /**
+     * Sets the version of the SDK package that generated this Event using the Packagist name.
+     *
+     * @param string $packageName
```

Comment:
```suggestion
     * @param string $packageName
     *
     * @internal
```
     Let's make these functions internal.